src/AspNetCore/Src/inprocessstoredcontext.cxx

```diff
@@ -32,6 +32,22 @@ IN_PROCESS_STORED_CONTEXT::QueryHttpContext(
     return m_pHttpContext;
 }
 
+VOID
+IN_PROCESS_STORED_CONTEXT::SetRequestAborted(
+    BOOL fRequestAborted
```

Comment:
since you cannot change a request  back to un-aborted if it got aborted, no need for passing in the bool value